packages/grafana-ui/src/components/Select/getSelectStyles.ts

```diff
@@ -27,6 +28,7 @@ export const getSelectStyles = stylesFactory((theme: GrafanaTheme2) => {
 
       '&:hover': {
         background: theme.colors.action.hover,
+        '& span': { textDecoration: 'underline' },
```

Comment:
We could consider removing this to differentiate between focused and hovered

Reply:
I think it looks pretty clear without the underline imo, having the underline might imply to some that an option is a link.

Reply:
big +1 to this, i don't think we should underline